src/OpenLoco/src/Vehicles/VehicleHead.cpp

```diff
@@ -874,7 +874,9 @@ namespace OpenLoco::Vehicles
     bool VehicleHead::updateLand()
     {
         Vehicle2* vehType2 = _vehicleUpdate_2;
-        if ((!(vehType2->var_73 & Flags73::isBrokenDown) || (vehType2->var_73 & Flags73::isStillPowered)) && status == Status::approaching)
+
+        // If dont have any running issue and is aproaching
+        if (vehType2->hasNo73Flags() && (status == Status::approaching))
```

Comment:
we don't know yet if isBrokenDown and isStillPowered is all of the 73 flags. Therefore this will have to stay as two has73 calls and a not or.

Reply:
Sure, will fix